static/elements/chromedash-guide-editall-page.js

```diff
@@ -135,6 +135,7 @@ export class ChromedashGuideEditallPage extends LitElement {
     return html`
       <form name="feature_form" method="POST" action="/guide/editall/${this.featureId}">
         <input type="hidden" name="token">
+        <input type="hidden" name"form_fields" value="edit_all">
```

Comment:
How is this supposed to work?   I was expecting the value to be a comma-separated list of field names.  I suppose you could have `touched()` interpret `"edit_all"` as a special value, but I don't see code for that.

Reply:
My bad, I had some misunderstanding of the code. I have made some changes, please review it again.

I have fixed it by passing the form fields along with the forms from the backend and creating a getFormFields() method to assemble a list of comma-separated list of field names for both verify_accuracy and edit_all pages. The unit tests are also modified to check if the form field values are correct.

This getFormFields() method may seem trivial but in the future it can be renamed as getEditedFormFields() and check what fields have been changed.